src/stringToMpdXml.js

```diff
@@ -7,9 +7,16 @@ export const stringToMpdXml = (manifestString) => {
   }
 
   const parser = new DOMParser();
-  const xml = parser.parseFromString(manifestString, 'application/xml');
-  const mpd = xml && xml.documentElement.tagName === 'MPD' ?
-    xml.documentElement : null;
+  let xml;
+  let mpd;
+
+  try {
+    xml = parser.parseFromString(manifestString, 'application/xml');
```

Comment:
IE 11 throws on invalid xml - which isn't illogical - it's just not the error that we want to throw. We want our custom `DASH_INVALID_XML` error.